src/sentry/static/sentry/app/components/featureBadge.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Tag from 'app/views/settings/components/tag';
+import Tooltip from 'app/components/tooltip';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import theme from 'app/utils/theme';
+import CircleIndicator from 'app/components/circleIndicator';
+
+type BadgeProps = {
+  type: 'alpha' | 'beta' | 'new';
+  variant?: 'indicator' | 'badge';
+  title?: string | null;
+};
+
+type Props = Omit<React.HTMLAttributes<HTMLDivElement>, keyof BadgeProps> & BadgeProps;
+
+const defaultTitles = {
+  alpha: t('This feature is in alpha and may be unstable'),
+  beta: t('This feature is in beta and may change in the future'),
+  new: t('This is a new feature'),
+};
+
+const labels = {
+  alpha: t('alpha'),
+  beta: t('beta'),
+  new: t('new'),
+};
+
+const FeaturedBadge = ({type, variant = 'badge', title, ...p}: Props) => {
+  const tooltipTitle = title === undefined ? defaultTitles[type] : title;
+
+  return (
+    <div {...p}>
+      <Tooltip title={tooltipTitle} disabled={!tooltipTitle} position="right">
+        <React.Fragment>
```

Comment:
The reason we have the fragment here is due to the child types on `Tooltip`. it doesn't like that the two nodes can resolve to `false`.